tests/jose/jws.spec.ts

```diff
@@ -7,35 +8,47 @@ import jwkSecp256k1Private from './vectors/jwk-secp256k1-private.json';
 chai.use(chaiAsPromised);
 
 describe('Jws', () => {
-  it('should sign and verify SECP256K1 signature using a key vector correctly',  async () => {
-    const publicKeyJwk = Object.assign({ }, jwkSecp256k1Private); // Clone private key.
-    delete publicKeyJwk.d; // Remove the private key portion.
+  it('should sign and verify secp256k1 signature using a key vector correctly',  async () => {
```

Comment:
This test appears to be broken running browser mode.

Reply:
Passing now! switching from using `Buffer` -> `Uint8Array` fixed the issue 🙃